readme.md

```diff
@@ -26,86 +23,31 @@ You need to deploy azure-log-forwarder in each region you want to pull logs from
-| --require-valid-certificate | REQUIRE_VALID_CERTIFICATE | Enables checking SSL certificate of the target Active Gate. By default (if this option is not provided) certificates aren't validated. | No | false |
-| --target-paas-token | TARGET_PAAS_TOKEN | Dynatrace PaaS token generated in Settings->Integration->Platform as a Service. Parameter required for deployment with containerized ActiveGate | Yes/No | - |
-| --target-api-token | TARGET_API_TOKEN | Dynatrace API token. You can learn how to generate token [Dynatrace API - Tokens and authentication](https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication) manually. **Integration requires 'API v2 Ingest logs' token permission.** | Yes | - |
-| --enable-self-monitoring | SFM_ENABLED | Send custom metrics to Azure. To use it remember to set up Managed identity for Function App (more details in Self-monitoring section). Self monitoring allows to diagnose quickly if your function processes and sends logs to Dynatrace properly. By default (if this option is not provided) custom metrics won't be sent to Azure. | No | false | 
-| --resource-group | RESOURCE_GROUP | Name of the Azure Resource Group in which Function will be deployed. To create new one run: `az group create --name <resource_group> --location <region>` | Yes | - |
-| --event-hub-connection-string | EVENT_HUB_CONNECTION_STRING | Connection string for Azure EventHub that is configured for receiving logs. You can create policy in Event Hub Namespace -> Event Hub -> Shared Access policies (listen permission is required). More info here: [https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string) | Yes | - |
-| --event-hub-name | EVENT_HUB_NAME | Name of Azure Event Hub configured for receiving logs  | Yes | - |
-| --filter-config | FILTER_CONFIG | Filter config applied as key-value pairs - more info in log filtering section | No | - |
-
-
-</center>
-
-### Update Dynatrace Log Forwarder code
-You can download latest azure-log-forwarder zip from:
-``` 
-wget https://github.com/dynatrace-oss/dynatrace-azure-log-forwarder/releases/latest/download/dynatrace-azure-log-forwarder.zip
-``` 
-To deploy new version of azure-log-forwarder run:
-```  
-az webapp deployment source config-zip -g <resource_group> -n <app_name> --src <zip_file_path>
-```
 
-### Uninstall Dynatrace Log Forwarder
-Deployment script tags all created resources with `LogsForwarderDeployment = {DEPLOYMENT_NAME}` tag. If you want to uninstall Dynatrace Log Forwarder, in Azure Portal navigate to Resource Group used for installation, filter resources by tag, and delete resources.
+##Issues 
 
 #### Virtual Network cleanup
```

Comment:
I would remove it from github doc, it's already in Dynatrace documentation in Troubleshoot section.

Reply:
+ it's worth to add Pricing section like in GCP doc (part with log ingest)

Reply:
Added